mono/metadata/class-init.c

```diff
@@ -3017,34 +3008,6 @@ mono_class_setup_vtable_general (MonoClass *klass, MonoMethod **overrides, int o
 
 	TRACE_INTERFACE_VTABLE (print_vtable_full (klass, vtable, cur_slot, first_non_interface_slot, "AFTER INHERITING PARENT VTABLE", TRUE));
 
-	/* Process overrides from interface default methods */
-	// FIXME: Ordering between interfaces
-	for (int ifindex = 0; ifindex < klass->interface_offsets_count; ifindex++) {
-		ic = klass->interfaces_packed [ifindex];
-
-		mono_class_setup_methods (ic);
-		if (mono_class_has_failure (ic))
-			goto fail;
-
-		MonoMethod **iface_overrides;
-		int iface_onum;
-		mono_class_get_overrides_full (ic->image, ic->type_token, &iface_overrides, &iface_onum, mono_class_get_context (ic), error);
-		goto_if_nok (error, fail);
-		for (int i = 0; i < iface_onum; i++) {
-			MonoMethod *decl = iface_overrides [i*2];
-			MonoMethod *override = iface_overrides [i*2 + 1];
-			if (decl->is_inflated) {
-				override = mono_class_inflate_generic_method_checked (override, mono_method_get_context (decl), error);
-				mono_error_assert_ok (error);
-			} else if (mono_class_is_gtd (override->klass)) {
-				override = mono_class_inflate_generic_method_full_checked (override, ic, mono_class_get_context (ic), error);
-			}
-			if (!apply_override (klass, ic, vtable, decl, override, &override_map, &override_class_map, &conflict_map))
-				goto fail;
-		}
-		g_free (iface_overrides);
-	}
-
```

Comment:
I don't understand why it's ok to move this later after the `/* override interface methods */` `for`-loop.  Doesn't that change which overrides get used?

Reply:
Yes, I think so, I just change to check if this would break any test. I think I should try to create a test that tests it.